hepcrawl/spiders/pos_spider.py

```diff
@@ -277,7 +278,7 @@ def build_conference_proceedings_item(
         record.add_value('journal_title', 'PoS')
         record.add_value(
             'journal_volume',
-            self._get_journal_volume(pos_id=pos_id),
+            self._get_journal_volume(identifier=pos_id),
```

Comment:
identifier is too generic, it's a pos internal identifier? a pos external one? orcid? inspire? what?